src/analysisd/decoders/winevtchannel.c

```diff
@@ -241,14 +206,24 @@ int DecodeWinevt(Eventinfo *lf){
                                 for (l = 0; child_attr[p]->attributes[l]; l++) {
                                     if (!strcmp(child_attr[p]->attributes[l], "Name") && strcmp(child_attr[p]->content, "(NULL)") != 0
                                             && strcmp(child_attr[p]->content, "-") != 0) {
-                                        filtered_string = replace_win_format(child_attr[p]->content);
+                                        filtered_string = replace_win_format(child_attr[p]->content,0);
                                         *child_attr[p]->values[l] = tolower(*child_attr[p]->values[l]);
-                                        cJSON_AddStringToObject(json_eventdata_in, child_attr[p]->values[l], filtered_string);
+
+                                        // Ignore category ID
+                                        if (!strcmp(child_attr[p]->values[l], "categoryId")){
```

Comment:
We cannot ignore these fields here, agents under 3.9 don't report the new fields in text format so they have to include these ones at least.

Reply:
This is the JSON event generated for a 3.8 agent with the new changes:

```
{"win":{"system":{"providerName":"Microsoft-Windows-Security-Auditing","providerGuid":"{54849625-5478-4994-A5BA-3E3B0328C30D}","eventID":"4719","version":"0","level":"0","task":"13568","opcode":"0","keywords":"0x8020000000000000","systemTime":"2019-04-01T15:06:36.122872000Z","eventRecordID":"15716035","processID":"564","threadID":"3528","channel":"Security","computer":"pcname","severityValue":"AUDIT_SUCCESS","message":"System audit policy was changed."},"eventdata":{"subjectUserSid":"S-1-5-18","subjectUserName":"PCNAME$","subjectDomainName":"WORKGROUP","subjectLogonId":"0x3e7","categoryId":"%%8273","subcategoryId":"%%12544","subcategoryGuid":"{0CCE9215-69AE-11D9-BED3-505054503030}","auditPolicyChanges":"%%8449"}}}
```
And the same event for a 3.9 agent:

```
{"win":{"system":{"providerName":"Microsoft-Windows-Security-Auditing","providerGuid":"{54849625-5478-4994-A5BA-3E3B0328C30D}","eventID":"4719","version":"0","level":"0","task":"13568","opcode":"0","keywords":"0x8020000000000000","systemTime":"2019-04-01T15:08:32.108760500Z","eventRecordID":"15716109","processID":"564","threadID":"3528","channel":"Security","computer":"pcname","severityValue":"AUDIT_SUCCESS","message":"System audit policy was changed."},"eventdata":{"subjectUserSid":"S-1-5-18","subjectUserName":"PCNAME$","subjectDomainName":"WORKGROUP","subjectLogonId":"0x3e7","subcategoryGuid":"{0CCE9215-69AE-11D9-BED3-505054503030}","category":"Logon/Logoff","subcategory":"Logon","changes":"Success Added"}}}
```